packages/components/src/components/data-list/data-list-row-wrapper.jsx

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+// eslint-disable-next-line react/display-name
+const DataListRowWrapper = React.memo(props => {
+    const [showDesc, setShowDesc] = React.useState(false);
+    const { destination_link, acion_desc, row_key, row_gap } = props;
+
+    return (
+        <div className='data-list__row--wrapper' style={{ paddingBottom: `${row_gap || 0}px` }}>
+            {destination_link ? (
+                <NavLink
+                    className='data-list__item--wrapper'
+                    id={`dt_reports_contract_${row_key}`}
+                    to={{
+                        pathname: destination_link,
+                        state: {
+                            from_table_row: true,
+                        },
+                    }}
+                >
+                    {props.children}
```

Comment:
`memo` and `children` don't work together. Refer [this thread](https://github.com/binary-com/deriv-app/pull/2183#discussion_r531025773) for more information